src/MaterialExtension.cpp

```diff
@@ -1,14 +1,17 @@
 #include "MaterialExtension.h"
+#include "MaterialParameters.h"
 
+using namespace MaterialParameters;
 
 MaterialExtension::MaterialExtension(const G4String& name, const G4Material* baseMaterial)
 : G4Material(name,baseMaterial->GetDensity(),baseMaterial,    
    baseMaterial->GetState(),baseMaterial->GetTemperature(),
    baseMaterial->GetPressure())
 {
-    foPsPobability = 0.;
     foPslifetime = 0.;
+    foPsFraction = 0.;
     fTarget = false ;
+    fPickOffFraction = 0.;
```

Comment:
add comments: why you use this this value as default and what does this variable represent,